tests/acceptance/customCommands/setValueBySingleKeys.js

```diff
@@ -10,12 +10,17 @@
  * @returns
  */
 exports.command = function setValueBySingleKeys (selector, inputValue) {
-  const chars = inputValue.split('')
-  if (chars.length === 0) {
-    return this.setValue(selector, '')
+  console.log('inputValue: ' + "'" + inputValue + "'")
+  if (inputValue) {
+    const chars = inputValue.split('').slice(0, -2)
+    const charsEnd = inputValue.split('').slice(-2)
+    const charPromise = chars.map((char) => this.setValue(selector, char)) || []
+
+    // Sometimes the autocomplete list is not displayed when the characters are entered very fast
+    // So we add a small pause for entering the last two characters
+    const charEndPromise = charsEnd.map((char) => this.setValue(selector, char).pause(100)) || []
```

Comment:
does this change has a connection with fedsharing? or is this for an other PR?

Reply:
This problem might only affect federation sharing as other tests were passing with no issue before this.